R/mod_utilization.R

```diff
@@ -0,0 +1,151 @@
+#' utilization UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+# mod_UI
+mod_utilization_ui <- function(id){
+  ns <- shiny::NS(id)
+  shiny::fluidPage(
+    ui_header_row(),
+    ui_row(box(
+      pickerInput(
```

Comment:
@trevinflick Can we use `ui_picker_program` here?

Reply:
Added!